python/lsst/ap/association/mapApData.py

```diff
@@ -250,15 +298,61 @@ def calibrateFluxes(self, inputRecord, outputRecord, photoCalib):
                 self.config.copyColumns[col_name + "_instFluxErr"],
                 meas.error)
 
+    def computeDipoleFluxes(self, inputRecord, outputRecord, photoCalib):
+        """Calibrate and compute dipole mean flux and diff flux.
+
+        Parameters
+        ----------
+        inputRecord : `lsst.afw.table.SourceRecord`
+            Record to copy flux values from.
+        outputRecord : `lsst.afw.table.SourceRecord`
+            Record to copy and calibrate values into.
+        photoCalib  `lsst.afw.image.PhotoCalib`
+            Calibration object from the difference exposure.
+        """
+
+        neg_meas = photoCalib.instFluxToNanojansky(
+            inputRecord, self.config.dipFluxPrefix + "_neg")
+        pos_meas = photoCalib.instFluxToNanojansky(
+            inputRecord, self.config.dipFluxPrefix + "_pos")
+        outputRecord.set(
+            "dipMeanFlux",
+            0.5 * (np.abs(neg_meas.value) + np.abs(pos_meas.value)))
+        outputRecord.set(
+            "dipMeanFluxErr",
+            0.5 * np.sqrt(neg_meas.error ** 2 + pos_meas.error ** 2))
+        outputRecord.set(
+            "dipFluxDiff",
+            np.abs(pos_meas.value) - np.abs(neg_meas.value))
+        outputRecord.set(
+            "dipFluxDiffErr",
+            np.sqrt(neg_meas.error ** 2 + pos_meas.error ** 2))
+
+    def computeDipoleSep(self, inputRecord, outputRecord, wcs):
+        """Calibrate and compute dipole mean flux and diff flux.
```

Comment:
Shouldn't this docstring be "compute dipole separation"?

Reply:
Thanks, for catching this. I copy pasted the previous method's docs and forgot to change the description.